doc/sphinx/language/extensions/match.rst

```diff
@@ -90,11 +90,15 @@ constructions. There are two variants of them.
 First destructuring let syntax
 ++++++++++++++++++++++++++++++
 
+.. todo explain that this applies to all of the "let" constructs (Gallina, Ltac1 and Ltac2)
+   also add "irrefutable pattern" to the glossary
+   note that an upper case ident is a constructor, lower case is a variable
```

Comment:
Isn't this last restriction only for Ltac2?

Reply:
Yup.